BUILD_ANDROID.md

```diff
@@ -74,12 +80,17 @@ The above code to suppress modules is not necessary, but will speed up the build
 
 * Open Android Studio
 * Choose _Open an existing Android Studio project_
-* Navigate to the `hifi` repository and choose the `android` folder and select _OK_
+* Navigate to the `project-athena` repository you cloned and choose the `android` folder and select _OK_
 * Wait for Gradle to sync (this should take around 20 minutes the first time)
-* From the _Build_ menu select _Make Project_
+* If a dialog pops open saying "Plugin Update Recommeded" dialog, do not click update, just click X on the top right to close.
+* In the _Projet_ window click on the project you wish to build (i.e. "questInterface") then click _Build_ in the top menu and choose _Make Module 'questInterface'_
+* By default this will build the "debug" apk, you can change this by opening the _Build Varients_ window along the left side and select other build types like "release"
```

Comment:
```suggestion
* By default this will build the "debug" apk, you can change this by opening the _Build Variants_ window along the left side and select other build types such as "release".
```